tests/unit/field/test_field3d.cxx

```diff
@@ -1778,6 +1779,74 @@ TEST_F(Field3DTest, DC) {
   EXPECT_TRUE(IsField2DEqualBoutReal(DC(field), 3.0));
 }
 
+TEST_F(Field3DTest, interp_to) {
```

Comment:
I'm not sure if we want to test all different locations in one test or to have a separate test for each,@ZedThree can probably comment more on this. The former saves duplicating setup etc., but the later helps more clearly isolate where the problem has occurred and if it occurs for multiple cases.

Reply:
Splitting them up would be better, for the reasons @d7919 says -- it makes it a bit clearer when there's a problem with one case vs multiple cases.

To save on the duplicating setup, I would put all the setup in a fixture -- copy/paste the `Field3DTest` class to `Field3DInterpToTest`, put all the setup in there (including the definition and creation of `input`, but not the definition of `output`), then make a bunch of tests like:

```cpp
TEST_F(Field3DInterpToTest, CellCentreToXLow) {
```
etc.

Reply:
Done. Is the way I set the class up correct? I had a bit of trouble initializing static members, etc. but I think I sorted it out.